src/SearchResults/index.js

```diff
@@ -740,42 +782,63 @@ SearchResults.prototype.getRefinements = function() {
     });
   });
 
-  forEach(state.tagRefinements, function(name) {
+  state.tagRefinements.forEach(function(name) {
     res.push({type: 'tag', attributeName: '_tags', name: name});
   });
 
   return res;
 };
 
+/**
+ * @typedef {Object} Facet
+ * @property {string} name
+ * @property {Object} data
+ * @property {boolean} exhaustive
+ */
+
+/**
+ * @param {*} state
+ * @param {*} type
+ * @param {string} attributeName
+ * @param {*} name
+ * @param {Facet[]} resultsFacets
+ */
 function getRefinement(state, type, attributeName, name, resultsFacets) {
   var facet = find(resultsFacets, {name: attributeName});
-  var count = get(facet, 'data[' + name + ']');
-  var exhaustive = get(facet, 'exhaustive');
   return {
     type: type,
     attributeName: attributeName,
     name: name,
-    count: count || 0,
-    exhaustive: exhaustive || false
+    count: (facet && facet.data[name]) || 0,
```

Comment:
I'm not sure if `data` is necessarily part of `facet` at this point. If it is, ignore the rest of this comment.

The exact equivalent is:

```suggestion
    count: (facet && facet.data && facet.data[name]) || 0,
```

I tried [Lodash `get` REPL](https://lodash.com/docs/4.17.11#get):

```js
const facetName = 'name'
const facet = {}

const res1 = _.get(facet, 'data[facetName]')
// => undefined

const res2 = facet && facet.data[facetName]
// => TypeError: Cannot read property 'name' of undefined

const res3 = facet && facet.data && facet.data[facetName]
// => undefined
```

Reply:
I think it might always be available, but let's be safe here indeed

Reply:
@Haroenv You did the change already?